docker/volumeplugin/rancher.go

```diff
@@ -17,6 +17,7 @@ const (
 )
 
 var goodStates = map[string]bool{
+	"inactive":     true,
```

Comment:
An also not created.  This will definitely break things.  If you removed inactive form goodStates, what fails to work?